ktor-network/jvm/src/io/ktor/network/sockets/SocketOptions.kt

```diff
@@ -145,6 +147,11 @@ sealed class SocketOptions(
          */
         var keepAlive: Boolean? = null
 
+        /**
+         * Socket timeout.
```

Comment:
Same here: we should document what kind of timeouts it is

Reply:
Updated.